launch-game.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+TEMPLATE_LAUNCHER=$(python -c "import os; print(os.path.realpath('$0'))")
+TEMPLATE_ROOT=$(dirname "${TEMPLATE_LAUNCHER}")
+source ${TEMPLATE_ROOT}/mod.config
+
+MOD_SEARCH_PATHS="./../.."

```

Comment:
This sets the mod search path to the parent directory of the RA2 clone.  This means that if you have a shared source directory (e.g. `~/src/` with subdirectories for e.g. `ra2` and `d2`) then this will pick up the other mods as well as ra2.  This will probably break things in unexpected ways.